### PR TITLE
Feature/add disable keep alives

### PR DESCRIPTION
**Pull request by fulldump**
feat: Add parameter to disable HTTP keep alives (-a and --disableKeepAlives)

    * It is only compatible with --http1 and --http2 (not with fasthttp)
    * Add `disableKeepAlives` (boolean) to args_parser.go and config.go
    * Set parameter to clientOpts (bombardier.go)
    * Configure http client transport with `DisableKeepAlives` (clients.go)

### CODE REVIEW

**Comment by codesenberg** [comment]
Thanks, @fulldump. Very cool.
I suggest that you rebase your commits on top the most recent commit in `release-1.2` branch and then we can start working on it, ok?

File: args_parser.go [hunk 1]
```diff
@@ -108,6 +109,10 @@ func newKingpinParser() argsParser {
 			" chain and host name").
 		Short('k').
 		BoolVar(&kparser.insecure)
+	app.Flag("disableKeepAlives",
+		"Disable HTTP KeepAlive").
```
Comment (codesenberg) [inline args_parser.go]: I think it should also mention that for `fasthttp` a header needs to be set. [See here.](https://github.com/valyala/fasthttp/issues/623#issuecomment-520124446)
Something like "Disable HTTP keep-alive. For fasthttp also set -H 'Connection: close'".

**Comment by codesenberg** [comment]
Shame this PR was stale for more than a year and small tweaks preventing it from being merged weren't resolved, but I added [this functionality anyway](https://github.com/codesenberg/bombardier/commit/c19a3b41dc634d3480dea1e21d2bb0ec97ea83d5). Wish there was some way to give you credit, but thanks for PR anyways 👍

**Comment by fulldump** [comment]
> Shame this PR was stale for more than a year and small tweaks preventing it from being merged weren't resolved, but I added [this functionality anyway](https://github.com/codesenberg/bombardier/commit/c19a3b41dc634d3480dea1e21d2bb0ec97ea83d5). Wish there was some way to give you credit, but thanks for PR anyways 👍

No worries at all, it is my bad. Btw, I still use bombardier whenever I need to test servers performance, it is a great tool!

**Pull request closed without merging**